Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -24,6 +24,8 @@
 
 #include "BaseLib/Algorithm.h"
 #include "BaseLib/FileTools.h"
+// include due to the forward declaration in ProcessVariable.h
+#include "BaseLib/TimeInterval.h"
```

Comment:
This looks to me to be at wrong place. It should be in `DeactivatedSubdomain.h` because of the unique_ptr used there; see https://stackoverflow.com/a/6089065

Reply:
took it.